src/bootstrap.jl

```diff
@@ -24,27 +24,18 @@ replicates: 1000
 function bootweights(design::SurveyDesign; replicates = 4000, rng = MersenneTwister(1234))
```

Comment:
Kind of unrelated, but why is the function named `bootweights`?  Could it be refactored to `ReplicateDesign(design::SurveyDesign, ...)`?  This would make the #292 approach a lot easier to implement.  If it's an R function you want to wrap, there is always `const bootweights = ReplicateDesign` or similar.

Reply:
There are several ways of making replicate weights. This is the first one that we have implemented. 

We may choose your approach, we are thinking about the design in #292 and a couple of other issues.